src/databases/OpenPMD/OpenPMDClasses/PMDFile.C

```diff
@@ -127,6 +128,13 @@ void PMDFile::OpenFile(char * PMDFilePath)
 
     H5Pclose(fileAccessPropListID);
 
+    // Check for error opening the file
+    if (fileId < 0)
+    {
+        EXCEPTION1(InvalidDBTypeException,
+                   "Not an openPMD file: Error opening the file");
+    }
```

Comment:
@brugger1 @dpugmire with this patch applied, somehow all openPMD HDF5 files are now reading for me:
```
Not an openPMD file: Error opening the file
```

Reply:
Update: it looks like some other process, likely an old `h5py`, had the file open (with "rw" by old default), that caused this.